### PR TITLE
Add factorybot and faker

### PR DESCRIPTION
**Pull request by aliuk2012**
#### What problem does the pull request solve?

We need factorybot & faker for testing and have been putting it off for a long period of time. This PR is to get factorybot & faker in and replaces a few tests to show the factories/faker is working but we should refactor as we work on new features.

Trello card:

#### Checklist

- [ ] I've used the pull request template
- [ ] I've linked this PR to the relevant issue (if mission work)
- [ ] I've written unit tests for these changes (if code change)
- [ ] I've updated the documentation in (If any documentation requires updating)
  - [ ] README.md
  - [ ] Elsewhere (please link)

### CODE REVIEW

File: spec/factories/models/forms.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+FactoryBot.define do
```
Comment (DavidBiddle) [inline spec/factories/models/forms.rb:1]: Not a comment for this PR but wonder if it's a) possible and b) desirable to share these factories between the repos

Comment (DavidBiddle) [reply spec/factories/models/forms.rb:1]: May also just be one for the microservices v monolith debate

**Review by DavidBiddle — DISMISSED** [review]
🎉

File: spec/factories/models/forms.rb (hunk 1 above)
Comment (aliuk2012) [reply spec/factories/models/forms.rb:1]: It is possible to share these factories in a gem, I do think that factories between forms-admin & forms-runner might be the same but forms-api could be different (ActiveRecord vs ActiveResource). Will take a deeper look when i get a moment.

**Review by DavidBiddle — APPROVED** [review]

**Review by cadmiumcat — DISMISSED** [review]

**Review by cadmiumcat — APPROVED** [review]

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**